ruby/README.md

```diff
@@ -49,12 +33,12 @@ Credentials Reference Guide](https://docs.aws.amazon.com/credref/latest/refdocs/
 * bundler 2.3.8 or later (to use bundler to install gems)
 
 ## Tests
+**Note**: Running the tests might result in charges to your AWS account.
 
-All tests use RSpec, and you can find them in the `spec` folder for each example.
-When an example has additional requirements to run tests, you can find them in the
-README for that service or cross-service example.
+All tests use RSpec, and you can find them in the `spec` folder for each example. Please check the README for each service or cross-service example for any additional requirements to run tests.
 
 ### Unit tests
+**Note**: Running the tests might result in charges to your AWS account.
```

Comment:
I always write unit tests against stubs or mocks, so unit tests never incur charges. But this may not be true across the board, so your call on whether to keep this.

Reply:
It's in the template, so I didn't want to delete it. I'll keep it until we update the template. Also added this to retro.